documentation/Deployment.md

```diff
@@ -0,0 +1,2 @@
+# Deployment Instructions
+Visit www.politech.xyz to use our web app. It is hosted with a GCP VM instance.
```

Comment:
Is the latest working code deployed yet? When I go there the first tab just seems to fire off infinite requests, and the other tabs give some Dash related errors.

Reply:
We're still working out some issues with the deployed version.  It doesn't seem to enjoy running 24/7 so we're looking at our code to see what could be causing errors after a while.  Rebooting fixes it so we've considered just writing a script to reboot the app every few hours.